### PR TITLE
Improve registrant change documentation

### PR DESCRIPTION
**Pull request by kntsoriano**
Belongs to https://github.com/dnsimple/dnsimple-business/issues/1709.

This PR improves the documentation based on the acceptance testing done.

### CODE REVIEW

File: content/v2/openapi.yml [hunk 1]
```diff
@@ -2776,6 +2781,27 @@ components:
...
+      in: query
+      required: false
+      description: Only include results with a state field exactly matching the given string
+      schema:
+        type: string
+    FilterDomainId:
+      name: domain
+      in: query
+      required: false
+      description: Only include results with a domain_id field exactly matching the given string
```
Comment (DXTimer) [inline content/v2/openapi.yml:2795]: The description does not match the filter name. `domain_id` vs `domain`

File: content/v2/openapi.yml [hunk 2]
```diff
@@ -2776,6 +2781,27 @@ components:
...
+      schema:
+        type: string
+    FilterContactId:
+      name: contact
+      in: query
+      required: false
+      description: Only include results with a contact_id field exactly matching the given string
```
Comment (DXTimer) [inline content/v2/openapi.yml:2802]: Same as the above. `contact_id` vs `contact`

File: content/v2/openapi.yml [hunk 3]
```diff
@@ -2776,6 +2781,27 @@ components:
       description: Only include results with a type field exactly matching the given string
       schema:
         type: string
+    FilterRegistrantChangeState:
+      name: state
+      in: query
+      required: false
+      description: Only include results with a state field exactly matching the given string
+      schema:
```
Comment (DXTimer) [inline content/v2/openapi.yml:2789]: Have you considered having this as an enum if the states are discrete?

**Review by DXTimer — APPROVED** [review]
Left some comments. Once those are addressed this is good to go. :shipit:

File: content/v2/openapi.yml (hunk 3 above)
Comment (kntsoriano) [reply content/v2/openapi.yml:2789]: Thanks :+1: I did this.

File: content/v2/openapi.yml (hunk 1 above)
Comment (kntsoriano) [reply content/v2/openapi.yml:2801]: Fixed :+1:

File: content/v2/openapi.yml (hunk 2 above)
Comment (kntsoriano) [reply content/v2/openapi.yml:2808]: Fixed :+1:

**Pull request merged**